presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -134,6 +139,22 @@ private void insertData(byte[] data)
         }
     }
 
+    private void createMessagesWithHeader(String topicName)
+    {
+        try (KafkaProducer<byte[], byte[]> producer = createProducer()) {
+            ProducerRecord<byte[], byte[]> record = new ProducerRecord<byte[], byte[]>(topicName, null, "{}".getBytes(Charset.forName("UTF-8")));
+            record.headers()
+                    .add("notfoo", null);
+            producer.send(record);
+            record = new ProducerRecord<byte[], byte[]>(topicName, null, "{}".getBytes(Charset.forName("UTF-8")));
+            record.headers()
+                    .add("foo", "bar".getBytes(Charset.forName("UTF-8")))
+                    .add("foo", null)
+                    .add("foo", "baz".getBytes(Charset.forName("UTF-8")));
+            producer.send(record);
```

Comment:
Can you also publish a message without headers?

Reply:
good point.